tools/kete.py

```diff
@@ -543,13 +543,19 @@ def _log_drawings_available(self, device):
 
 
 class Fetcher(Worker):
-    def __init__(self, manager, args):
+    def __init__(self, manager, args, config):
         super(Fetcher, self).__init__(manager)
         self.device = None
         self.timestamps = None
         address = args.address
         index = args.index
 
+        if not address in config:
+            config[address] = {}
+
+        self.orientation = int(config[address].get('Orientation', 0))
+        self.handle_pressure = int(config[address].get('HandlePressure', 0))
```

Comment:
Ideally I would prefer this to be a boolean, and use `config[address].getboolean('HandlePressure', False)` instead